From 8d5de4985be45f5fa6235cb289bc538b4f3d004d Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sun, 26 Oct 2025 14:36:32 -0500
Subject: [PATCH] Include user unit

---
 xdg-desktop-portal-cosmic.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xdg-desktop-portal-cosmic.spec b/xdg-desktop-portal-cosmic.spec
index 6fe8631..6353fec 100644
--- a/xdg-desktop-portal-cosmic.spec
+++ b/xdg-desktop-portal-cosmic.spec
@@ -98,6 +98,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %license cargo-vendor.txt
 %{_libexecdir}/xdg-desktop-portal-cosmic
 %{_datadir}/dbus-1/services/org.freedesktop.impl.portal.desktop.cosmic.service
+%{_userunitdir}/org.freedesktop.impl.portal.desktop.cosmic.service
 %{_datadir}/xdg-desktop-portal/portals/cosmic.portal
 %{_datadir}/xdg-desktop-portal/cosmic-portals.conf
 %{_datadir}/icons/hicolor/scalable/actions/screenshot-screen-symbolic.svg
-- 
2.51.0

