From 8915b761a13902ec9a18b105479d446b92aae08d Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 27 Oct 2025 22:53:58 +0000
Subject: [PATCH] Include systemd-rpm-macros

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 xdg-desktop-portal-cosmic.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xdg-desktop-portal-cosmic.spec b/xdg-desktop-portal-cosmic.spec
index 6fe8631..bd08303 100644
--- a/xdg-desktop-portal-cosmic.spec
+++ b/xdg-desktop-portal-cosmic.spec
@@ -45,6 +45,7 @@ BuildRequires:  mesa-libgbm-devel
 BuildRequires:  gstreamer1-devel
 BuildRequires:  glib2-devel
 BuildRequires:  just
+BuildRequires:  systemd-rpm-macros
 
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
 
-- 
2.51.0

