diff -ruN ./vendor/pop-launcher/debian/pop-launcher.install ./vendor/pop-launcher.new/debian/pop-launcher.install
--- ./vendor/pop-launcher/debian/pop-launcher.install	2024-09-29 17:03:28.707579672 -0500
+++ ./vendor/pop-launcher.new/debian/pop-launcher.install	2024-09-29 17:07:19.159431963 -0500
@@ -1,3 +1,3 @@
 /usr/bin/
-/usr/lib/pop-launcher/plugins/
-/usr/lib/pop-launcher/scripts/session/
\ No newline at end of file
+/usr/libexec/pop-launcher/plugins/
+/usr/libexec/pop-launcher/scripts/session/
\ No newline at end of file
diff -ruN ./vendor/pop-launcher/debian/pop-launcher.links ./vendor/pop-launcher.new/debian/pop-launcher.links
--- ./vendor/pop-launcher/debian/pop-launcher.links	2024-09-29 17:03:28.707579672 -0500
+++ ./vendor/pop-launcher.new/debian/pop-launcher.links	2024-09-29 17:07:19.167432027 -0500
@@ -1,11 +1,11 @@
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/calc/calc
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/desktop_entries/desktop-entries
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/files/files
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/find/find
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/pop_shell/pop-shell
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/pulse/pulse
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/recent/recent
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/scripts/scripts
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/terminal/terminal
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/web/web
-/usr/bin/pop-launcher /usr/lib/pop-launcher/plugins/cosmic_toplevel/cosmic-toplevel
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/calc/calc
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/desktop_entries/desktop-entries
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/files/files
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/find/find
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/pop_shell/pop-shell
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/pulse/pulse
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/recent/recent
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/scripts/scripts
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/terminal/terminal
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/web/web
+/usr/bin/pop-launcher /usr/libexec/pop-launcher/plugins/cosmic_toplevel/cosmic-toplevel
diff -ruN ./vendor/pop-launcher/debian/pop-launcher-system76-power.install ./vendor/pop-launcher.new/debian/pop-launcher-system76-power.install
--- ./vendor/pop-launcher/debian/pop-launcher-system76-power.install	2024-09-29 17:03:28.707579672 -0500
+++ ./vendor/pop-launcher.new/debian/pop-launcher-system76-power.install	2024-09-29 17:07:19.159431963 -0500
@@ -1 +1 @@
-/usr/lib/pop-launcher/scripts/system76-power
\ No newline at end of file
+/usr/libexec/pop-launcher/scripts/system76-power
\ No newline at end of file
diff -ruN ./vendor/pop-launcher/debian/rules ./vendor/pop-launcher.new/debian/rules
--- ./vendor/pop-launcher/debian/rules	2024-09-29 17:03:28.702579632 -0500
+++ ./vendor/pop-launcher.new/debian/rules	2024-09-29 17:07:19.159431963 -0500
@@ -32,4 +32,4 @@
 
 override_dh_fixperms:
 	dh_fixperms
-	chmod +x debian/pop-launcher/usr/lib/pop-launcher/scripts/**/*.sh
\ No newline at end of file
+	chmod +x debian/pop-launcher/usr/libexec/pop-launcher/scripts/**/*.sh
\ No newline at end of file
diff -ruN ./vendor/pop-launcher/README.md ./vendor/pop-launcher.new/README.md
--- ./vendor/pop-launcher/README.md	2024-09-29 17:03:28.689579527 -0500
+++ ./vendor/pop-launcher.new/README.md	2024-09-29 17:07:19.156431939 -0500
@@ -45,7 +45,7 @@
 
 - User-local plugins: `~/.local/share/pop-launcher/plugins/{plugin}/`
 - System-wide install for system administrators: `/etc/pop-launcher/plugins/{plugin}/`
-- Distribution packaging: `/usr/lib/pop-launcher/plugins/{plugin}/`
+- Distribution packaging: `/usr/libexec/pop-launcher/plugins/{plugin}/`
 
 ## Plugin Config
 
@@ -79,7 +79,7 @@
 
 - User-local scripts: `~/.local/share/pop-launcher/scripts`
 - System-wide install for system administrators: `/etc/pop-launcher/scripts`
-- Distribution packaging: `/usr/lib/pop-launcher/scripts`
+- Distribution packaging: `/usr/libexec/pop-launcher/scripts`
 
 Example script
 <details>
diff -ruN ./vendor/pop-launcher/src/lib.rs ./vendor/pop-launcher.new/src/lib.rs
--- ./vendor/pop-launcher/src/lib.rs	2024-09-29 17:03:28.690579535 -0500
+++ ./vendor/pop-launcher.new/src/lib.rs	2024-09-29 17:07:19.159431963 -0500
@@ -19,7 +19,7 @@
 pub const SYSTEM: &str = "/etc/pop-launcher";
 pub const SYSTEM_PLUGINS: &str = concatcp!(SYSTEM, "/plugins");
 
-pub const DISTRIBUTION: &str = "/usr/lib/pop-launcher";
+pub const DISTRIBUTION: &str = "/usr/libexec/pop-launcher";
 pub const DISTRIBUTION_PLUGINS: &str = concatcp!(DISTRIBUTION, "/plugins");
 
 pub const PLUGIN_PATHS: &[&str] = &[LOCAL_PLUGINS, SYSTEM_PLUGINS, DISTRIBUTION_PLUGINS];
