From be161bd1963e0a3f3b96ddacb1ac5cb1b0eb6e68 Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Wed, 6 May 2026 16:53:59 +0200
Subject: [PATCH] remove files for cosmic-notif

---
 cosmic-notifications.spec | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/cosmic-notifications.spec b/cosmic-notifications.spec
index 798c25a..ff07d48 100644
--- a/cosmic-notifications.spec
+++ b/cosmic-notifications.spec
@@ -36,7 +36,6 @@ BuildRequires:  cargo
 BuildRequires:  wayland-devel
 BuildRequires:  libxkbcommon-devel
 BuildRequires:  just
-BuildRequires:  desktop-file-utils
 
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
 
@@ -77,7 +76,6 @@ export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
 %check
-desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicNotifications.desktop
 %if %{with check}
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
@@ -91,9 +89,6 @@ export VERGEN_GIT_SHA="%{commit}"
 %license cargo-vendor.txt
 %doc README.md
 %{_bindir}/cosmic-notifications
-%{_datadir}/applications/com.system76.CosmicNotifications.desktop
-%{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicNotifications.svg
-%{_metainfodir}/com.system76.CosmicNotifications.metainfo.xml
 
 %changelog
 %autochangelog
-- 
2.54.0

