From e6c8f2847a3267d03767f146d12f89252618bc73 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:46:07 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-notifications.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-notifications.spec b/cosmic-notifications.spec
index 7d95158..b18fdeb 100644
--- a/cosmic-notifications.spec
+++ b/cosmic-notifications.spec
@@ -75,16 +75,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicNotifications.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicNotifications.desktop
 %if %{with check}
-- 
2.50.1

