From c62b52446b80169adf0627fcb913d356de2f915c Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Tue, 25 Nov 2025 17:36:40 -0600
Subject: [PATCH] Use just instead of make

---
 cosmic-osd.spec | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cosmic-osd.spec b/cosmic-osd.spec
index 717f115..290cc0c 100644
--- a/cosmic-osd.spec
+++ b/cosmic-osd.spec
@@ -44,7 +44,7 @@ BuildRequires:  systemd-devel
 BuildRequires:  pulseaudio-libs-devel
 BuildRequires:  pipewire-devel
 BuildRequires:  clang
-BuildRequires:  make
+BuildRequires:  just
 
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
 Requires:       pulseaudio-utils
@@ -72,7 +72,7 @@ fi
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
-make all polkit-agent-helper-1=/usr/lib/polkit-1/polkit-agent-helper-1
+just polkit-agent-helper-1=/usr/lib/polkit-1/polkit-agent-helper-1 build-release --offline --frozen
 %{cargo_license_summary}
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
@@ -83,7 +83,7 @@ sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' -i cargo-vendor.txt
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
-make install DESTDIR=%{buildroot} prefix=%{_prefix}
+just rootdir=%{buildroot} prefix=%{_prefix} install
 
 %if %{with check}
 %check
-- 
2.51.1

