From e0bd9ab4634a3fd541e43e3bc356eb27d6bdf417 Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Wed, 3 Sep 2025 09:35:32 +0200
Subject: [PATCH] add pipewire and pulse

---
 cosmic-osd.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cosmic-osd.spec b/cosmic-osd.spec
index 099f6b2..372367a 100644
--- a/cosmic-osd.spec
+++ b/cosmic-osd.spec
@@ -40,9 +40,12 @@ BuildRequires:  wayland-devel
 BuildRequires:  libxkbcommon-devel
 BuildRequires:  systemd-devel
 BuildRequires:  pulseaudio-libs-devel
+BuildRequires:  pipewire-devel
+BuildRequires:  clang
 BuildRequires:  make
 
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
+Requires:       pulseaudio-utils
 
 %global _description %{expand:
 %{summary}.}
-- 
2.51.0

