From 433dd4db3aa018bfce7d6395224f1605ff8099ee Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-osd.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-osd.spec b/cosmic-osd.spec
index 099f6b2..cf5f23c 100644
--- a/cosmic-osd.spec
+++ b/cosmic-osd.spec
@@ -72,6 +72,7 @@ make all polkit-agent-helper-1=/usr/lib/polkit-1/polkit-agent-helper-1
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

