From 29850a64dde204e201ae93bb0131e302e0e732ae Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 22 Sep 2025 02:56:17 +0000
Subject: [PATCH] Add new required dependencies

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-osd.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cosmic-osd.spec b/cosmic-osd.spec
index 099f6b2..e307cb5 100644
--- a/cosmic-osd.spec
+++ b/cosmic-osd.spec
@@ -37,6 +37,8 @@ BuildRequires:  rustc
 BuildRequires:  lld
 BuildRequires:  cargo
 BuildRequires:  wayland-devel
+BuildRequires:  libinput-devel
+BuildRequires:  clang-devel
 BuildRequires:  libxkbcommon-devel
 BuildRequires:  systemd-devel
 BuildRequires:  pulseaudio-libs-devel
-- 
2.51.0

