From ac2df8d74444106f35301bc5f2e447496f14838f Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 13 Dec 2025 12:19:54 -0600
Subject: [PATCH] Add patch to fix ppc64le

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-randr.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cosmic-randr.spec b/cosmic-randr.spec
index 7d4d353..e55bbb2 100644
--- a/cosmic-randr.spec
+++ b/cosmic-randr.spec
@@ -29,6 +29,9 @@ Source1:        vendor-%{version_no_tilde}.tar.gz
 # * mv vendor-config-%%{version_no_tilde}.toml ..
 Source2:        vendor-config-%{version_no_tilde}.toml
 
+# https://github.com/pop-os/cosmic-randr/pull/51
+Patch: https://patch-diff.githubusercontent.com/raw/pop-os/cosmic-randr/pull/51.patch
+
 BuildRequires:  cargo-rpm-macros >= 25
 BuildRequires:  rustc
 BuildRequires:  lld
-- 
2.51.1

