From eabdf59db5da09e18e128c57ed0629a11e51cb4c Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:45:43 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-app-library.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-app-library.spec b/cosmic-app-library.spec
index 495a1a7..e88741d 100644
--- a/cosmic-app-library.spec
+++ b/cosmic-app-library.spec
@@ -76,16 +76,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicAppLibrary.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicAppLibrary.desktop
 %if %{with check}
-- 
2.50.1

