From 11aab6d9d58f37d85c1c91d9a26a5f6b2f30308a Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:47:52 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-app-library.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-app-library.spec b/cosmic-app-library.spec
index 495a1a7..c8420bf 100644
--- a/cosmic-app-library.spec
+++ b/cosmic-app-library.spec
@@ -69,6 +69,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

