From 6626b01db4c8b06283f86226d65fd465026392f3 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-panel.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-panel.spec b/cosmic-panel.spec
index cba50e2..e3d68e3 100644
--- a/cosmic-panel.spec
+++ b/cosmic-panel.spec
@@ -65,6 +65,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' -i cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

