From a53ce6fe6685d9882e7cf1ff92003463ff6ee93b Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 26 Apr 2025 14:33:27 -0500
Subject: [PATCH 1/2] Fix building for s390x and ppc64le

---
 cosmic-edit.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cosmic-edit.spec b/cosmic-edit.spec
index 1075b41..7cecf67 100644
--- a/cosmic-edit.spec
+++ b/cosmic-edit.spec
@@ -4,7 +4,7 @@ ExcludeArch: %{ix86}
 %bcond_without check
 
 %global crate cosmic-edit
-
+%global build_rustflags %{?build_rustflags} --cfg=io_uring_skip_arch_check
 
 # While our version corresponds to an upstream tag, we still need to define
 # these macros in order to set the VERGEN_GIT_SHA and VERGEN_GIT_COMMIT_DATE
-- 
2.49.0

