From ceeaeff41cd8d30fb288e19eecbe342cd66ffcd4 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:45:56 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-edit.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-edit.spec b/cosmic-edit.spec
index 7cecf67..2ca346a 100644
--- a/cosmic-edit.spec
+++ b/cosmic-edit.spec
@@ -76,16 +76,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicEdit.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicEdit.desktop
 %if %{with check}
-- 
2.50.1

