From 80ea576db26c9526c3b0cc25df9c61fc79e068b8 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 2 Mar 2026 21:10:18 -0600
Subject: [PATCH] Build with the packagekit feature

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-store.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cosmic-store.spec b/cosmic-store.spec
index b02ca18..f13f596 100644
--- a/cosmic-store.spec
+++ b/cosmic-store.spec
@@ -68,7 +68,7 @@ fi
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
-%cargo_build
+%{cargo_build} --features packagekit
 %{cargo_license_summary}
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
@@ -88,7 +88,7 @@ just rootdir=%{buildroot} prefix=%{_prefix} install
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
-%cargo_test
+%{cargo_test} --features packagekit
 %endif
 
 %files
-- 
2.53.0

