From c4b29aadf546955fdc4c22feb6e384f656716f2d Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:46:19 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-store.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-store.spec b/cosmic-store.spec
index b0d01b6..67ea085 100644
--- a/cosmic-store.spec
+++ b/cosmic-store.spec
@@ -80,16 +80,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicStore.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicStore.desktop
 %if %{with check}
-- 
2.50.1

