From 3084cb8cfdcc9b0e8bb7b64bb734fe3e9e2d8315 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-session.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-session.spec b/cosmic-session.spec
index d25b30d..e981ea7 100644
--- a/cosmic-session.spec
+++ b/cosmic-session.spec
@@ -99,6 +99,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

