From d51ce08b1eb7e87179edcafef663c4ce4aa31565 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Fri, 10 Jul 2026 20:27:43 -0500
Subject: [PATCH] Add libdav1d-devel for avif

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-settings.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-settings.spec b/cosmic-settings.spec
index 90315f6..3fedc43 100644
--- a/cosmic-settings.spec
+++ b/cosmic-settings.spec
@@ -43,6 +43,7 @@ BuildRequires:  just
 BuildRequires:  desktop-file-utils
 BuildRequires:  pipewire-devel
 BuildRequires:  clang
+BuildRequires:  libdav1d-devel
 
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
 Requires:       nm-connection-editor
-- 
2.54.0

