From cc65ae23a901d406b5c87017b0e6bc86cc0aae09 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Fri, 10 Jul 2026 22:09:08 -0500
Subject: [PATCH] Add theme v2 directories

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-settings.spec | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cosmic-settings.spec b/cosmic-settings.spec
index 3fedc43..28ca33b 100644
--- a/cosmic-settings.spec
+++ b/cosmic-settings.spec
@@ -132,12 +132,16 @@ export VERGEN_GIT_SHA="%{commit}"
 %{_metainfodir}/com.system76.CosmicSettings.metainfo.xml
 %dir %{_datadir}/cosmic/com.system76.CosmicTheme.Dark.Builder
 %{_datadir}/cosmic/com.system76.CosmicTheme.Dark.Builder/v1/*
+%{_datadir}/cosmic/com.system76.CosmicTheme.Dark.Builder/v2/*
 %dir %{_datadir}/cosmic/com.system76.CosmicTheme.Dark
 %{_datadir}/cosmic/com.system76.CosmicTheme.Dark/v1/*
+%{_datadir}/cosmic/com.system76.CosmicTheme.Dark/v2/*
 %dir %{_datadir}/cosmic/com.system76.CosmicTheme.Light.Builder
 %{_datadir}/cosmic/com.system76.CosmicTheme.Light.Builder/v1/*
+%{_datadir}/cosmic/com.system76.CosmicTheme.Light.Builder/v2/*
 %dir %{_datadir}/cosmic/com.system76.CosmicTheme.Light
 %{_datadir}/cosmic/com.system76.CosmicTheme.Light/v1/*
+%{_datadir}/cosmic/com.system76.CosmicTheme.Light/v2/*
 %dir %{_datadir}/cosmic/com.system76.CosmicTheme.Mode
 %{_datadir}/cosmic/com.system76.CosmicTheme.Mode/v1/*
 %dir %{_datadir}/cosmic/com.system76.CosmicComp
-- 
2.54.0

