From 6b7c4c439dcf06dfa1da733dedc9b7d60ea021ef Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:54:41 -0500
Subject: [PATCH 2/2] Validate cosmic-settings page desktop files

---
 cosmic-settings.spec | 35 +++++++++++++++++------------------
 1 file changed, 17 insertions(+), 18 deletions(-)

diff --git a/cosmic-settings.spec b/cosmic-settings.spec
index 6b33fc1..27527c2 100644
--- a/cosmic-settings.spec
+++ b/cosmic-settings.spec
@@ -97,24 +97,23 @@ just rootdir=%{buildroot} install
 
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.desktop
-# TODO: Fix desktop file validation of sub pages upstream
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.About.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Appearance.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.DateTime.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Dock.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Panel.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.WindowManagement.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Displays.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Firmware.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Keyboard.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Mouse.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Notifications.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.RegionLanguage.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Sound.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Touchpad.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Users.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Wallpaper.desktop
-# desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Workspaces.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.About.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Appearance.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.DateTime.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Dock.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Panel.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.WindowManagement.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Displays.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Firmware.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Keyboard.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Mouse.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Notifications.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.RegionLanguage.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Sound.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Touchpad.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Users.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Wallpaper.desktop
+desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicSettings.Workspaces.desktop
 %if %{with check}
 # Set vergen environment variables
 export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
-- 
2.50.1

