From 899983c9ea03091f5ce9bfdcceb7fe19c24241fb Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-idle.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-idle.spec b/cosmic-idle.spec
index d22ce96..ad0600a 100644
--- a/cosmic-idle.spec
+++ b/cosmic-idle.spec
@@ -70,6 +70,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' -i cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

