From 333e92f7cb8a3601eea4f711427decc2e339521b Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:47:52 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-greeter.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-greeter.spec b/cosmic-greeter.spec
index 4a706a4..1ffcccb 100644
--- a/cosmic-greeter.spec
+++ b/cosmic-greeter.spec
@@ -85,6 +85,7 @@ cargo build --release --all --offline --frozen
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

