From 03a60dcb3de84735c312d1375adca2e390ffd1f2 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 6 Jul 2026 21:24:31 -0500
Subject: [PATCH] Update license

---
 cosmic-greeter.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cosmic-greeter.spec b/cosmic-greeter.spec
index 41acd1f..9b803de 100644
--- a/cosmic-greeter.spec
+++ b/cosmic-greeter.spec
@@ -17,7 +17,7 @@ Release:        %autorelease
 # Release:        %%autorelease
 Summary:        Login and display manager for the COSMIC Desktop Environment
 
-License:        (0BSD OR Apache-2.0 OR MIT) AND Apache-2.0 AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR CC0-1.0 OR MIT) AND (Apache-2.0 OR MIT) AND (Apache-2.0 OR MIT OR Zlib) AND BSD-2-Clause AND BSD-3-Clause AND BSL-1.0 AND CC-PDDC AND CC0-1.0 AND GPL-3.0-only AND ISC AND MIT AND (MIT OR Unlicense) AND MPL-2.0 AND Unicode-3.0 AND Zlib
+License: (0BSD OR Apache-2.0 OR MIT) AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSD-2-Clause OR MIT) AND (Apache-2.0 OR BSD-3-Clause OR MIT) AND (Apache-2.0 OR BSD-3-Clause) AND (Apache-2.0 OR CC0-1.0 OR MIT) AND (Apache-2.0 OR CC0-1.0) AND (Apache-2.0 OR GPL-2.0-only) AND (Apache-2.0 OR LGPL-2.1-or-later OR MIT) AND (Apache-2.0 OR MIT OR Zlib) AND (Apache-2.0 OR MIT) AND (LGPL-3.0-or-later OR MIT) AND (MIT OR Unlicense) AND Apache-2.0 AND BSD-2-Clause AND BSD-3-Clause AND BSL-1.0 AND CC-PDDC AND CC0-1.0 AND GPL-3.0-only AND ISC AND MIT AND MPL-2.0 AND NCSA AND Unicode-3.0 AND Zlib
 
 URL:            https://github.com/pop-os/cosmic-greeter
 
-- 
2.54.0

