From 54ccddd304548364bbb8aa491796a79368a31319 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:46:25 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-workspaces.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-workspaces.spec b/cosmic-workspaces.spec
index b67db64..885e861 100644
--- a/cosmic-workspaces.spec
+++ b/cosmic-workspaces.spec
@@ -85,16 +85,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 make install DESTDIR=%{buildroot} prefix=%{_prefix}
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicWorkspaces.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicWorkspaces.desktop
 %if %{with check}
-- 
2.50.1

