From d30227ca56a47283f7a27c9dd6b2603c18fc280b Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Sat, 19 Jul 2025 02:33:53 +0200
Subject: [PATCH] add thumbailer for cosmic player

---
 cosmic-player.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-player.spec b/cosmic-player.spec
index 20e935d..5ad4964 100644
--- a/cosmic-player.spec
+++ b/cosmic-player.spec
@@ -110,6 +110,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{_datadir}/applications/com.system76.CosmicPlayer.desktop
 %{_metainfodir}/com.system76.CosmicPlayer.metainfo.xml
 %{_datadir}/icons/hicolor/*/apps/com.system76.CosmicPlayer.svg
+%{_datadir}/thumbnailers/com.system76.CosmicPlayer.thumbnailer
 
 %changelog
 %autochangelog
-- 
2.50.1

