From 212a6468904a3eaaff806bad017b9aa2bae5c8a9 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:46:10 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-player.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-player.spec b/cosmic-player.spec
index 20e935d..8d8ac44 100644
--- a/cosmic-player.spec
+++ b/cosmic-player.spec
@@ -83,16 +83,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicPlayer.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicPlayer.desktop
 %if %{with check}
-- 
2.50.1

