From 7206dec6186fa3417a90604173979947acbe26c0 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 14 Jan 2026 12:43:50 -0600
Subject: [PATCH] Add cmake as a build dependency

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-initial-setup.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-initial-setup.spec b/cosmic-initial-setup.spec
index b976cf0..483e337 100644
--- a/cosmic-initial-setup.spec
+++ b/cosmic-initial-setup.spec
@@ -49,6 +49,7 @@ BuildRequires:  openssl-devel
 BuildRequires:  flatpak-devel
 BuildRequires:  libinput-devel
 BuildRequires:  systemd-devel
+BuildRequires:  cmake
 
 Recommends:     PackageKit
 Recommends:     flatpak
-- 
2.51.1

