From bb81d40b8d65fb8d55937087db97b6dabdbeca00 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sun, 16 Nov 2025 23:40:36 -0600
Subject: [PATCH] Add gvfs modules

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-files.spec | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/cosmic-files.spec b/cosmic-files.spec
index f0e5baf..28564d0 100644
--- a/cosmic-files.spec
+++ b/cosmic-files.spec
@@ -42,6 +42,11 @@ BuildRequires:  libxkbcommon-devel
 BuildRequires:  desktop-file-utils
 
 Requires:       gvfs
+# GVFS backends
+Requires:       gvfs-fuse
+Requires:       gvfs-nfs
+Requires:       gvfs-smb
+
 Requires:       xdg-utils
 Requires:       cosmic-icon-theme >= %{cosmic_minver}
 
-- 
2.51.1

