From eab21846cfb1853b632679606eb6f486f70f080f Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:45:58 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-files.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-files.spec b/cosmic-files.spec
index ca77308..54bfb06 100644
--- a/cosmic-files.spec
+++ b/cosmic-files.spec
@@ -82,16 +82,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicFiles.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicFiles.desktop
 %if %{with check}
-- 
2.50.1

