From a446de1c0096f8ad049a067269ae00a6a8c40794 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 22 Sep 2025 11:32:57 -0500
Subject: [PATCH] Add patch to fix cosmic-files

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-files.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cosmic-files.spec b/cosmic-files.spec
index ca77308..f38e73a 100644
--- a/cosmic-files.spec
+++ b/cosmic-files.spec
@@ -32,6 +32,9 @@ Source1:        vendor-%{version_no_tilde}.tar.gz
 # * mv vendor-config-%%{version_no_tilde}.toml ..
 Source2:        vendor-config-%{version_no_tilde}.toml
 
+# Fixes desktop file
+# https://github.com/pop-os/cosmic-files/pull/1196
+Patch: https://patch-diff.githubusercontent.com/raw/pop-os/cosmic-files/pull/1196.patch
 
 BuildRequires:  cargo-rpm-macros >= 26
 BuildRequires:  rustc
-- 
2.50.1

