From 7cd0c631f3ae00660a65d57918709df17bdfc97b Mon Sep 17 00:00:00 2001
From: Adil Hanney <adilhanney@disroot.org>
Date: Sun, 8 Mar 2026 21:50:30 +0000
Subject: [PATCH] Add Qt theming dependencies

---
 cosmic-settings-daemon.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cosmic-settings-daemon.spec b/cosmic-settings-daemon.spec
index 5721c8d..b23e2fc 100644
--- a/cosmic-settings-daemon.spec
+++ b/cosmic-settings-daemon.spec
@@ -50,8 +50,11 @@ BuildRequires:  openssl-devel
 Requires:       acpid
 Requires:       adw-gtk3-theme
 Requires:       pop-sound-theme
+Requires:       qt6ct
+Recommends:     breeze-icon-theme
 Recommends:     geoclue2
 Recommends:     playerctl
+Recommends:     qt5ct
 
 %global _description %{expand:
 %{summary}.}
-- 
2.53.0

