From c1aee359f349011d45a7d36b4fb3a197f527d00f Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Thu, 18 Jun 2026 12:36:00 +0200
Subject: [PATCH] add pipewire to settings daemon

---
 cosmic-settings-daemon.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-settings-daemon.spec b/cosmic-settings-daemon.spec
index 9ac8d9b..6767e1a 100644
--- a/cosmic-settings-daemon.spec
+++ b/cosmic-settings-daemon.spec
@@ -44,6 +44,7 @@ BuildRequires:  clang-devel
 BuildRequires:  libxkbcommon-devel
 BuildRequires:  pam-devel
 BuildRequires:  pulseaudio-utils
+BuildRequires:  pipewire-devel
 BuildRequires:  just
 BuildRequires:  openssl-devel
 
-- 
2.54.0

