From 654bfff0b7419c6c22bfe2f7117c31794d626393 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-settings-daemon.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-settings-daemon.spec b/cosmic-settings-daemon.spec
index edd0707..a01ce9e 100644
--- a/cosmic-settings-daemon.spec
+++ b/cosmic-settings-daemon.spec
@@ -80,6 +80,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

