From b246baf6cde34e1785f286d6e950afdb16c6c525 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-term.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-term.spec b/cosmic-term.spec
index bb8ac35..79c4cdf 100644
--- a/cosmic-term.spec
+++ b/cosmic-term.spec
@@ -70,6 +70,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

