From 76193ba33d480d9f5a953bcaa4e164ec7fa967aa Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 17 Sep 2025 12:46:02 -0500
Subject: [PATCH] Remove Fedora 40 workarounds

---
 cosmic-launcher.spec | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/cosmic-launcher.spec b/cosmic-launcher.spec
index a3b52df..2fa4180 100644
--- a/cosmic-launcher.spec
+++ b/cosmic-launcher.spec
@@ -79,16 +79,6 @@ export VERGEN_GIT_COMMIT_DATE="date --utc '%{commitdatestring}'"
 export VERGEN_GIT_SHA="%{commit}"
 just rootdir=%{buildroot} prefix=%{_prefix} install
 
-# COSMIC is not a valid category pre-fedora 41
-%if %{defined fedora} && 0%{?fedora} < 41
-desktop-file-install \
---remove-category COSMIC \
---add-category X-COSMIC \
---delete-original \
---dir %{buildroot}%{_datadir}/applications \
-%{buildroot}%{_datadir}/applications/com.system76.CosmicLauncher.desktop
-%endif
-
 %check
 desktop-file-validate %{buildroot}%{_datadir}/applications/com.system76.CosmicLauncher.desktop
 %if %{with check}
-- 
2.50.1

