From 16d17bdf2cb795bc5d06398bffb41361af361d2b Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Sat, 4 Oct 2025 13:08:44 +0200
Subject: [PATCH] add epoch

---
 pop-launcher.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pop-launcher.spec b/pop-launcher.spec
index 75ee911..03043e3 100644
--- a/pop-launcher.spec
+++ b/pop-launcher.spec
@@ -14,6 +14,7 @@ ExcludeArch: %{ix86}
 
 Name:           pop-launcher
 Version: 1.0.0~beta.1
+Epoch: 1
 Release:        %autorelease
 Summary:        Modular IPC-based desktop launcher service
 
-- 
2.51.0

