From ad78bff4a1b841a21be8ba10e777353282769df9 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Sat, 20 Sep 2025 00:49:29 -0500
Subject: [PATCH] license: Remove extra data from license versions

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 pop-launcher.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pop-launcher.spec b/pop-launcher.spec
index 9bd0be1..8fcb152 100644
--- a/pop-launcher.spec
+++ b/pop-launcher.spec
@@ -73,6 +73,7 @@ just build-release --offline --frozen
 %{cargo_license} > LICENSE.dependencies
 %{cargo_vendor_manifest}
 sed 's/\(.*\) (.*#\(.*\))/\1+git\2/' -i cargo-vendor.txt
+sed 's/^\([^+]*\)+.*+\([^+]*\)$/\1+\2/' cargo-vendor.txt
 
 %install
 # Set vergen environment variables
-- 
2.51.0

