From 501d490be77688e77b6b1994c691a6a3169bad24 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 24 Sep 2025 21:27:03 -0500
Subject: [PATCH] Use epoch of 1

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 pop-launcher.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pop-launcher.spec b/pop-launcher.spec
index 8fcb152..ae99276 100644
--- a/pop-launcher.spec
+++ b/pop-launcher.spec
@@ -15,6 +15,7 @@ ExcludeArch: %{ix86}
 Name:           pop-launcher
 Version: 1.2.4~alpha.7^git%{commitdate}.%{shortcommit}
 Release:        %autorelease
+Epoch: 1
 Summary:        Modular IPC-based desktop launcher service
 
 License:        (0BSD OR Apache-2.0 OR MIT) AND Apache-2.0 AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR ISC OR MIT) AND (Apache-2.0 OR MIT) AND (Apache-2.0 OR MIT OR Zlib) AND Apache-2.0 WITH LLVM-exception AND BSD-3-Clause AND GPL-3.0-only AND ISC AND MIT AND (MIT OR Unlicense) AND MPL-2.0 AND Zlib
-- 
2.51.0

