From 73a0c81eb33f29cdf9428f1350bc5821fbe59b51 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Tue, 7 Jul 2026 13:34:03 -0500
Subject: [PATCH] Update license

---
 pop-launcher.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pop-launcher.spec b/pop-launcher.spec
index 0329a0c..e975d0a 100644
--- a/pop-launcher.spec
+++ b/pop-launcher.spec
@@ -18,7 +18,7 @@ Epoch: 1
 Release:        %autorelease
 Summary:        Modular IPC-based desktop launcher service
 
-License:        (0BSD OR Apache-2.0 OR MIT) AND Apache-2.0 AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR ISC OR MIT) AND (Apache-2.0 OR MIT) AND (Apache-2.0 OR MIT OR Zlib) AND Apache-2.0 WITH LLVM-exception AND BSD-3-Clause AND GPL-3.0-only AND ISC AND MIT AND (MIT OR Unlicense) AND MPL-2.0 AND Zlib
+License: (0BSD OR Apache-2.0 OR MIT) AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR MIT OR Zlib) AND (Apache-2.0 OR MIT) AND (MIT OR Unlicense) AND MIT AND MPL-2.0 AND Unicode-3.0 AND Zlib
 
 URL:            https://github.com/pop-os/launcher
 
-- 
2.54.0

