From 8a74a9b8ffd03c8017ec4b887bc78cbea0192a6a Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Sun, 28 Jun 2026 23:44:09 +0200
Subject: [PATCH] add dbus policy file

---
 cosmic-applets.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cosmic-applets.spec b/cosmic-applets.spec
index 19a14d9..65cdd64 100644
--- a/cosmic-applets.spec
+++ b/cosmic-applets.spec
@@ -139,6 +139,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{_bindir}/cosmic-applet-workspaces
 %{_userunitdir}/com.system76.CosmicStatusNotifierWatcher.service
 %{_datadir}/dbus-1/services/com.system76.CosmicStatusNotifierWatcher.service
+%{_datadir}/dbus-1/system.d/com.system76.CosmicSettings.Applet.NetworkManager.SecretAgent.conf
 %{_datadir}/applications/com.system76.CosmicAppList.desktop
 %{_datadir}/applications/com.system76.CosmicAppletA11y.desktop
 %{_datadir}/applications/com.system76.CosmicAppletAudio.desktop
-- 
2.54.0

