From d4865a4aa900e386b36536410ffd19e54ee5a37d Mon Sep 17 00:00:00 2001
From: wiiznokes <78230769+wiiznokes@users.noreply.github.com>
Date: Thu, 21 Aug 2025 13:31:55 +0200
Subject: [PATCH] add pipewire

---
 cosmic-applets.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cosmic-applets.spec b/cosmic-applets.spec
index a9b0e8a..5aaa42c 100644
--- a/cosmic-applets.spec
+++ b/cosmic-applets.spec
@@ -45,6 +45,8 @@ BuildRequires:  libudev-devel
 BuildRequires:  pulseaudio-libs-devel
 BuildRequires:  mesa-libEGL-devel
 BuildRequires:  libxkbcommon-devel
+BuildRequires:  pipewire-devel
+BuildRequires:  clang
 BuildRequires:  just
 BuildRequires:  desktop-file-utils
 
-- 
2.50.1

