From 18a3c375b94cda5608f5f7b9291b386cbbaa220b Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Mon, 6 Jul 2026 21:24:52 -0500
Subject: [PATCH] Update license

---
 cosmic-screenshot.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cosmic-screenshot.spec b/cosmic-screenshot.spec
index 660dca5..a94d95f 100644
--- a/cosmic-screenshot.spec
+++ b/cosmic-screenshot.spec
@@ -19,7 +19,7 @@ Version: 1.1.0
 Release:        %autorelease
 Summary:        Screenshot utility for the COSMIC Desktop Environment
 
-License:        (0BSD OR Apache-2.0 OR MIT) AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR MIT) AND (Apache-2.0 OR MIT OR Zlib) AND BSD-3-Clause AND MIT AND (MIT OR Unlicense) AND MPL-2.0
+License: (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSD-2-Clause OR MIT) AND (Apache-2.0 OR GPL-2.0-only) AND (Apache-2.0 OR LGPL-2.1-or-later OR MIT) AND (Apache-2.0 OR MIT) AND (MIT OR Unlicense) AND MIT AND MPL-2.0 AND Unicode-3.0 AND Zlib
 
 URL:            https://github.com/pop-os/cosmic-screenshot
 
-- 
2.54.0

